src/App.js

```diff
@@ -1,12 +1,320 @@
+                                    name="customDelivery"
+                                    id="custom-delivery"
+                                    ref={register({
+                                        required: {
+                                            value: true,
+                                            message: 'U heeft geen alternative bezorgfrequentie aangegeven.'
+                                        }
+                                    })}
+                                />
+                            )}
+                        </div>
+                        <div className="delivery-message">
+                            <h3>Opmerking</h3>
+                            <textarea
+                                name="message"
+                                id="delivery-message"
+                                placeholder="Vul hier uw opmerking in (max. 200 karakters)."
+                                ref={register({
+                                        maxLength: {
+                                            value: 200,
+                                            message: 'Maximaal 200 karakters toegestaan.',
+                                        }
+                                    }
+                                )}
+                            >
+                            </textarea>
+                            {errors.message && <p className="error-message">{errors.message.message}</p>}
+                        </div>
+                    </fieldset>
+                    <label htmlFor="delivery-terms">
+                        <input
+                            type="checkbox"
+                            name="acceptedTerms"
+                            id="delivery-terms"
+                            ref={register({
+                                required: {
+                                    value: true,
+                                    message: 'U heeft de algemene voorwaarden niet geaccepteerd.'
+                                }
+                            })}
+                        />
+                        Ik ga akkoord met de algemene voorwaarden
+                    </label>
+                    <div className="error-block">
+                        {errors.name && <p className="error-message">{errors.name.message}</p>}
+                        {errors.surname && <p className="error-message">{errors.surname.message}</p>}
+                        {errors.age && <p className="error-message">{errors.age.message}</p>}
+                        {errors.zipcode && <p className="error-message">{errors.zipcode.message}</p>}
+                        {errors.houseNumber && <p className="error-message">{errors.houseNumber.message}</p>}
+                        {errors.customDelivery && <p className="error-message">{errors.customDelivery.message}</p>}
+                        {errors.acceptedTerms && <p className="error-message">{errors.acceptedTerms.message}</p>}
```

Comment:
Ik had elke error bij het desbetreffende veld gezet, maar dat is maar een minor detail 😉